packages/@tailwindcss-postcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -475,6 +475,14 @@ exports[`\`@import 'tailwindcss'\` is replaced with the generated CSS 1`] = `
     color: inherit;
   }
 
+  ::file-selector-button:where() {
```

Comment:
This is definitely wrong. Maybe this is a lightningcss bug? Looks like it works in the playground so we should probably work on upgrading.

Maybe split the selector in preflight temporarily to work around it?

Reply:
Ah yeah good catch. Noticing other stuff that's kinda in this same category of fixes, like `input` elements get a different background color when disabled too. Not sure how far to go with this stuff, tricky.